### PR TITLE
Misc makefile cleanups

### PR DESCRIPTION
**Pull request by joestringer**
Tidy up some make output and factor out a target to make local development a little easier.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6733)

### CODE REVIEW

**Review by ianvernon — APPROVED** [review]

File: envoy/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Authors of Cilium
+# Copyright 2017-2019 Authors of Cilium
```
Comment (aanm) [inline envoy/Makefile:1]: Do we still need to have this file? :thinking:

**Review by aanm — APPROVED** [review]

File: envoy/Makefile (hunk 1 above)
Comment (joestringer) [reply envoy/Makefile:1]: @jrajahalme ^

**Comment by joestringer** [comment]
test-me-please

**Comment by joestringer** [comment]
Failing only on ginkgo tests is a good sign, because that means that all the make targets worked. Marking as `ready-to-merge`.

File: envoy/Makefile (hunk 1 above)
Comment (vadorovsky) [reply envoy/Makefile:1]: I think we need it. See the `install` target at the end of this file. It takes care of unpacking `cilium-envoy` binary from the Docker image and installing it.

**Review by vadorovsky — APPROVED** [review]

**Comment by vadorovsky** [comment]
test-me-please

**Pull request merged**